install_build_dependencies.sh

```diff
@@ -95,6 +95,10 @@ elif [ -f /etc/redhat-release ]; then
             gcc \
             gcc-c++ \
             make \
+            patchelf \
+            pkg-config \
+            gflags-devel.i686 \
+            zlib-devel.i686 \
```

Comment:
I suppose gflags and zlib are not needed in 2022.1?

Reply:
Yes, unused gflags and zlib dependency was removed. Thanks.